orca-keel/src/main/kotlin/com/netflix/spinnaker/orca/keel/task/ImportDeliveryConfigTask.kt

```diff
@@ -153,6 +160,62 @@ constructor(
       ?: ""}/${context.manifest}@${context.ref}"
   }
 
+  /**
+   * Handle (potentially) retryable failures for SpinnakerServerException.
+   */
+  private fun handleRetryableFailures(serverException: SpinnakerServerException, context: ImportDeliveryConfigContext): TaskResult{
```

Comment:
I don't see this method getting called, though I think things would be simpler and more correct if it did.  Like, instead of catching SpinnakerNetworkException and SpinnakerHttpException above, catch SpinnakerServerException instead and call this method....and then either combine the network and http cases here, or call those methods based on the class of exception.

Reply:
Simplified the catch block with the parent exception class SpinnakerServerException. 
Introduced a new method which would handle all Spinnaker*Exception and smart casts on http error and/or network error, and also returns a default error message on other Spinnaker Exceptions.